hack/tests/origin-conformance.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash -ex
+
+PATH=`go env GOPATH`/bin::$PATH
+TEST_SUITE="${TEST_SUITE:-openshift/conformance/parallel/minimal}"
+
+cleanup() {
+  set +e
+
+  generate_artifacts
+
+  delete
+}
+
+fetch_origin() {
+  local branch="$1"
+  local remote="https://github.com/openshift/origin.git"
+
+  test -d /tmp/origin || git clone --depth=1 --branch=${branch} $remote /tmp/origin
```

Comment:
What if `/tmp/origin` exists but the repo is out of date? Or it contains something totally unexpected? I think it might make more sense to create a new temporary directory for the clone on each test run.

Reply:
This already runs in an ephemeral container in CI, so the checkout wouldn't exist in CI. I actually used this while testing because I could just use my local clone of origin mounted on /tmp/origin and avoid doing clones every single time.

Reply:
Eh, fair enough. I like things to be safe by default to reduce shooting oneself in the foot, but I suppose running this continuously locally should be approached with caution :)